packages/commonwealth/server/routes/communities/create_community_handler.ts

```diff
@@ -14,9 +16,27 @@ export const createCommunityHandler = async (
   req: TypedRequestBody<CreateCommunityRequestBody>,
   res: TypedResponse<CreateCommunityResponse>,
 ) => {
+  for (const key in req.body) {
+    if (req.body[key] === '' || req.body[key] === null) {
+      delete req.body[key];
+    }
+  }
+
+  const validationResult = await createCommunitySchema().safeParseAsync(
+    req.body,
+  );
+
+  if (validationResult.success === false) {
+    throw new AppError(
+      validationResult.error.issues
+        .map(({ path, message }) => `${path.join(': ')}: ${message}`)
+        .join(', '),
+    );
+  }
+
```

Comment:
Yeah thats a good point. Just that some error messages are meant to appear in the error toast (they need to look good since they are propagated directly to the user), and some would be meant to be handled on the backend (They are in a json format). So I guess we should make 2 methods, one for each type of expected end user of the error (front end devs or customers).

Reply:
Anyways I have abstracted this type of error formatting out. But after a quick look through our knowledge_base files I am not sure where to document it. Any ideas @gdjohnson?

Reply:
@kurtisassad What do you think of Code-Style.md? Alternatively we could start an Errors & Validation entry? Feel free to counterpropose.